plugins/onepad/onepad.cpp

```diff
@@ -286,7 +286,6 @@ PADfreeze(int mode, freezeData *data)
         strncpy(pdata->format, "OnePad", sizeof(pdata->format));
         pdata->version = PAD_SAVE_STATE_VERSION;
         pdata->query = query;
-
```

Comment:
I think it was better with the extra line here but maybe that's just me and my preference.